### ISSUE TITLE
NYC 3D Tiles demo doesn't work in 2D or CV

### ISSUE THREAD

**Issue by mramato** [AnalyticalGraphicsInc/cesium#5955]
http://localhost:8080/Apps/Sandcastle/index.html?src=3D%20Tiles%20Feature%20Picking.html&label=All

In 2D, buildings are way off.  In CV they are off and/or crooked depending on if you visited the tile in 3D mode first.

2D
![image](https://user-images.githubusercontent.com/1495196/32291920-d4f1f1ee-bf14-11e7-9dde-4ded0728ab2e.png)

CV

![image](https://user-images.githubusercontent.com/1495196/32291981-04aafa0c-bf15-11e7-8fd1-1932a6c7e373.png)

**Comment by mramato** [comment]
CC @lilleyse @pjcozzi

**Comment by pjcozzi** [comment]
Also, switch back to 3D and only a subset of the buildings stream in.  Perhaps there is an issue with what mode the scene is in when a tile is loaded?

![image](https://user-images.githubusercontent.com/782098/34044072-937c1f30-e171-11e7-8939-64d4301c5d22.png)

**Comment by mramato** [comment]
Do we think this and #6056 have the same root cause?

**Comment by pjcozzi** [comment]
Yes.

via @Vineg in #6056

> We have a 3dTiles model which gets corrupted after switching to 2d mode and then back to 3d. Some DrawCommands seem to have wrong BoundingSphere center and not being rendered.
Can you take a look at https://github.com/AnalyticalGraphicsInc/cesium/blame/b0939bf96697ec8c51cc32b8b96c5ed79b94a71a/Source/Scene/Model.js#L3798
It seems that when we switch back to 3d, model._rtcCenter not gets switched back to _rtcCenter3D

**Comment by OmarShehata** [comment]
Came up again [on the forum](https://groups.google.com/d/msg/cesium-dev/1HgKTXp6VrM/HMypdjRoCwAJ):

> In addition - i found that changing the view corrupts my data. this means, when I changed to 2d view and after that back to 3d view nothing will be displayed - "the data are lost"

**Comment by kteneyck** [comment]
Any update on these issues when switching to 2D with 3D Tiles loaded into Cesium? The only way we're going to be able to support 2D is to unload and reload any tilesets we have when switching views.

**Comment by lilleyse** [comment]
@kteneyck we're actively working on this and other 2D bugs: https://github.com/CesiumGS/cesium/issues/9934

**Comment by lilleyse** [comment]
@j9liu can this be closed now that https://github.com/CesiumGS/cesium/pull/10384 is merged?

**Comment by mramato** [comment]
@j9liu No, it wasn't. But if that's the case then I'm not sure why we would close this until everything "just works". I would consider this issue as part of the final testing that should happen when ModelExperimental goes away and just becomes Model.
